ClearKeep/Scenes/Search/Views/SearchAllView.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  SearchAllView.swift
+//  ClearKeep
+//
+//  Created by MinhDev on 05/04/2022.
+//
+
+import SwiftUI
+import Combine
+import Common
+import CommonUI
+
+private enum Constants {
+	static let paddingLeading = 100.0
+	static let spacing = 10.0
+	static let padding = 20.0
+	static let spacingContent = 20.0
+	static let paddingCatalog = 5.0
+	static let radius = 8.0
+	static let heightCatalog = 28.0
+}
+
+struct SearchAllView: View {
+	// MARK: - Constants
+	@Environment(\.colorScheme) var colorScheme
+	// MARK: - Variables
+	@Environment(\.injected) private var injected: DIContainer
+	@Binding var imageUser: Image
+	@Binding var userName: String
+	@Binding var message: String
+	@Binding var groupText: String
+	@Binding var dateMessage: String
+
+	// MARK: - Init
+	init(imageUser: Binding<Image>,
+		 userName: Binding<String>,
+		 message: Binding<String>,
+		 groupText: Binding<String>,
+		 dateMessage: Binding<String>) {
+		self._imageUser = imageUser
+		self._userName = userName
+		self._message = message
+		self._groupText = groupText
+		self._dateMessage = dateMessage
+	}
+
+	// MARK: - Body
+	var body: some View {
+		content
+			.padding(.horizontal, Constants.spacing)
+	}
+}
+
+// MARK: - Private
+private extension SearchAllView {
+	var content: AnyView {
+		AnyView(allView)
+	}
+}
+
+// MARK: - Private variable
+private extension SearchAllView {
+	var foregroundColorTitle: Color {
+		colorScheme == .light ? AppTheme.shared.colorSet.grey1 : AppTheme.shared.colorSet.greyLight
+	}
+
+	var backgroundButtonDark: LinearGradient {
+		LinearGradient(gradient: Gradient(colors: AppTheme.shared.colorSet.gradientPrimary), startPoint: .leading, endPoint: .trailing)
+	}
+
+	var backgroundButtonLight: LinearGradient {
+		LinearGradient(gradient: Gradient(colors: [AppTheme.shared.colorSet.offWhite, AppTheme.shared.colorSet.offWhite]), startPoint: .leading, endPoint: .trailing)
+	}
+}
+
+// MARK: - Private
+private extension SearchAllView {
+
+}
+
+// MARK: - Loading Content
+private extension SearchAllView {
+	var allView: some View {
+		ScrollView(showsIndicators: false) {
```

Comment:
Not need separate view if it is static view. ( Not contains other state like loading, error ...)

Reply:
fix complete